.gitmodules

```diff
@@ -7,3 +7,18 @@
 [submodule "lib/permit2"]
 	path = lib/permit2
 	url = git@github.com:Uniswap/permit2.git
+[submodule "lib/uniswapV2-zksync"]
```

Comment:
Is it possible to use the existing repos, or at least move this to the `contracts/test` directory?